tensorflow/tsl/platform/default/build_config.bzl

```diff
@@ -862,14 +862,44 @@ def tf_google_mobile_srcs_no_runtime():
 def tf_google_mobile_srcs_only_runtime():
     return []
 
+def if_llvm_aarch32_available(then, otherwise = []):
+    return select({
+        clean_dep("//tensorflow/tsl:linux_armhf"): then,
+        "//conditions:default": otherwise,
+    })
+
 def if_llvm_aarch64_available(then, otherwise = []):
-    return then
+    return select({
+        clean_dep("//tensorflow/tsl:linux_aarch64"): then,
```

Comment:
Aren't we missing Mac ARM here (and Mac x86 below as well)?

Reply:
Indeed, missed that sorry. I'll upload a new version.